src/main/java/hudson/plugins/parameterizedtrigger/BuildInfoExporterAction.java

```diff
@@ -129,6 +156,111 @@ public BuildReference(final String projectName) {
       this.buildNumber = 0;
       this.buildResult = Result.NOT_BUILT;
     }
+
+    public String getProjectName() {
+      return projectName;
+    }
+
+    public int getBuildNumber() {
+      return buildNumber;
+    }
+
+    public Result getBuildResult() {
+      return buildResult;
+    }
+
+    @Override
+    public void update() {
+
+    }
+  }
+
+  public static class StaticBuildReference extends AbstractBuildReference {
```

Comment:
@kwhetstone 
Please have a look whether the new solution looks better for you. I removed all the old fields and implemented a xstream-ConverterImpl handling the conversion.
Should I just leave the old name of BuildReference? Probably it's not worth renaming it just to have a "better" name describing that it's not changing once created. What do you think?